client/test/e2e/purchases/purchases.spec.js

```diff
@@ -0,0 +1,126 @@
+/* global element, by, browser */
+const chai = require('chai');
+const expect = chai.expect;
+
+const helpers = require('../shared/helpers');
+helpers.configure(chai);
+
+const FU = require('../shared/FormUtils');
+const PurchaseOrderPage = require('./purchases.page.js');
+const components = require('../shared/components');
+
+/*
+ * Simple Tests for Purchase Orders
+ */
+describe('Purchase Orders', function () {
+  'use strict';
+
+  const path = '#/purchases/new';
+
+  // navigate to the patient invoice page
+  beforeEach(() => helpers.navigate(path));
+
+  it('supports single item purchase orders', () => {
+    var page = new PurchaseOrderPage();
+
+    // prepare the page with default supplier, description, etc
+    FU.typeahead('PurchaseCtrl.supplier', 'Test Supplier');
+    FU.input('PurchaseCtrl.order.details.note', 'This is a brief description of what is going on');
+    components.dateEditor.set(new Date('2016-03-03'));
+
+    // set the 'other' delivery method parameter
+    $('#other').click();
```

Comment:
I think this is relevant to this discussion 
https://github.com/IMA-WorldHealth/bhima-2.X/pull/531#discussion_r68577726

Just for reference in the future!